src/Autocomplete/src/Form/AutocompleteChoiceTypeExtension.php

```diff
@@ -104,6 +148,7 @@ public function configureOptions(OptionsResolver $resolver): void
             'min_characters' => null,
             'max_results' => 10,
             'preload' => 'focus',
+            'extra_options' => [],
```

Comment:
Commenting there but it's more general... Don't you force the extra_options existence everywhere by setting it to [] instead of null ?

Reply:
AFAIR it'll work exactly the same way.